resources/js/Pages/Auth/NewTeam.vue

```diff
@@ -0,0 +1,83 @@
+<template>
+    <Layout>
+        <div class="bg-[#F9FAFB] h-screen w-screen flex items-center">
+            <div class="h-max mx-auto flex flex-col items-center">
+                <h1 class="text-xl font-bold text-center pb-10">
+                    Create your first team
+                </h1>
+                <form @submit.prevent="submit">
+                    <div
+                        class="bg-white shadow-xl p-10 flex flex-col gap-4 text-sm"
+                    >
+                        <div>
+                            <label
+                                class="text-gray-600 font-bold inline-block pb-2"
+                                for="name"
+                                >Team Name</label
+                            >
+                            <input
+                                v-model="form.name"
+                                class="border border-gray-400 focus:outline-slate-400 rounded-md w-full shadow-sm px-5 py-2"
+                                type="text"
+                                name="team_name"
+                                placeholder="My Team ..."
+                            />
+                            <div
+                            v-if="form.errors.name"
+                            v-text="form.errors.name"
+                            class="text-sm text-red-500 py-2"
+                        ></div>
+                        </div>
+                        <div>
+                            <label
+                                class="text-gray-600 font-bold inline-block pb-2"
+                                for="logo"
+                                >Team Logo</label
+                            >
+                            <input
+                                @input="form.logo = $event.target.files[0]"
```

Comment:
I know that the ticket says input type file but you could also use Dropzone Vue for better user experience rather than the input type file.